src/styles/components/card.scss

```diff
@@ -0,0 +1,101 @@
+@import "./utilits/colors.scss";
+
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white-color;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-left: 19px;
+    margin-right: 19px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    height: 100%;
+    img {
+      width: 160px;
+      height: 134px;
+      display: block;
+      width: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &__title {
+    color: $text-color;
+    font-family: 'Roboto', sans-serif;
```

Comment:
The main font-family should be applied to the body or html element.
Also, checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/5fdf71e0-01fd-4ae1-b88c-07c1b6e322b7)